### PR TITLE
feat: Added Lands Support

### PR DESCRIPTION
**Pull request by CoasterFreakDE**
Added Support for [Lands](https://www.spigotmc.org/resources/lands-%E2%AD%95-land-claim-plugin-%E2%9C%85-grief-prevention-protection-gui-management-nations-wars-1-20-support.53313/) - A towny-like claim Plugin

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
     <groupId>me.bestemor.villagermarket</groupId>
     <artifactId>VillagerMarket</artifactId>
-    <version>1.11.7-SNAPSHOT</version>
+    <version>1.11.8-SNAPSHOT</version>
```
Comment (Bestem0r) [inline pom.xml:14]: Next version will be 1.12.0. Can you change it?

**Review by Bestem0r — APPROVED** [review]
Great! Thanks a lot for your contribution

**Review by CoasterFreakDE — COMMENTED** [review]
chore: Update to 1.12.0

File: pom.xml (hunk 1 above)
Comment (CoasterFreakDE) [reply pom.xml]: Done.

**Pull request merged**